pages/map.py

```diff
@@ -236,7 +238,10 @@ def create_user_modes_options(trips_group_by_user_mode):
     Input('user-id-dropdown', 'value'),
 )
 def update_user_ids_options(trips_data, selected_user_ids):
-    user_ids_options, user_ids = create_user_ids_options(trips_data['users_data_by_user_id'])
+    user_ids_options, user_ids = list(), set()
+    if 'users_data_by_user_id' in trips_data:
+        user_ids_options, user_ids = create_user_ids_options(trips_data['users_data_by_user_id'])
+
```

Comment:
given that the issue is with the `mode_confirm` column being missing, do you even need these other checks?

Note that if `users_data_by_user_id` is not in trips_data, we return empty values, which may hide other errors (like the one with the missing mode_confirm)

Reply:
@shankari 
Cleaned it and checked the local is running well!

What I understand is, if `users_data_by_user_id` is not present in `trips_data`, we need to address this issue. So this line might obscure certain error cases.